File: app/src/main/java/com/example/devicemanager/activity/AddDeviceActivity.java
package com.example.devicemanager.activity;

import android.app.AlertDialog;
import android.content.DialogInterface;
import android.content.Intent;
import android.os.Bundle;
import android.text.Html;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.devicemanager.R;
import com.example.devicemanager.fragment.AddDeviceFragment;
import com.example.devicemanager.manager.Contextor;

public class AddDeviceActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_add_device);

        initInstances();

        if (savedInstanceState == null) {

            Bundle bundle = new Bundle();

            bundle.putString("Path", getIntent().getStringExtra("Path"));
            bundle.putString("Serial", getIntent().getStringExtra("Serial"));
            AddDeviceFragment fragment = AddDeviceFragment.newInstances();
            fragment.setArguments(bundle);

            getSupportFragmentManager().beginTransaction()
                    .add(R.id.contentContainer, fragment)
                    .commit();
        }
    }

    private void initInstances() {
        getSupportActionBar().setHomeButtonEnabled(true);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setTitle(Html.fromHtml("<font color='#ffffff'>Add Device</font>"));
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/example/devicemanager/activity/AddDeviceActivity.kt
package com.example.devicemanager.activity

import android.os.Bundle
import android.text.Html
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity

import com.example.devicemanager.R
import com.example.devicemanager.fragment.AddDeviceFragment

class AddDeviceActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_add_device)

        initInstances()

        if (savedInstanceState == null) {

            val bundle = Bundle()

            bundle.putString("Path", intent.getStringExtra("Path"))
            bundle.putString("Serial", intent.getStringExtra("Serial"))
            val fragment = AddDeviceFragment.newInstances()
            fragment.arguments = bundle

            supportFragmentManager.beginTransaction()
                    .add(R.id.contentContainer, fragment)
                    .commit()
        }
    }

    private fun initInstances() {
        supportActionBar!!.setHomeButtonEnabled(true)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.title = Html.fromHtml("<font color='#ffffff'>Add Device</font>")
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }
}
